src/cmds/index.js

```diff
@@ -28,9 +29,14 @@ const cli = () => {
 
             break;
         case 'login':
-            logger.info('@NOTE: Only user password auth is supported ATM.');
```

Comment:
thoughts of doing another prompt here - or providing option of which type of login ?

Reply:
hmmm I'm okay just leaving as is since we're only going to have two types.

Reply:
we could tackle this on a different ticket?